index.html

```diff
@@ -1,657 +1,1338 @@
+                              <img src="./assets/insta.svg" class="cmp-image__image" itemprop="contentUrl" data-cmp-hook-image="image" alt="instagram">
+                            </div>
+                          </div>
+
+                          <div class="image aem-GridColumn aem-GridColumn--default--12 layout-grid">
+                            <div data-cmp-lazythreshold="0" data-cmp-src="/content/leaderimpact/lang/en2/_jcr_content/content-parsys/container_428510884/container/container/container/container/image_1887433809.coreimg{.width}.svg/1669130230195/yt.svg" data-asset="/content/dam/LeaderImpact/site-./assets/yt.svg" data-title="YouTube" id="image-46d1bf6845" class="cmp-image" itemscope="" itemtype="http://schema.org/ImageObject">
+                              <img src="./assets/yt.svg" class="cmp-image__image" itemprop="contentUrl" data-cmp-hook-image="image" alt="YouTube">
+                            </div>
+                          </div>
+
+                          <div class="image aem-GridColumn aem-GridColumn--default--12 layout-grid">
+                            <div data-cmp-lazythreshold="0" data-cmp-src="/content/leaderimpact/lang/en2/_jcr_content/content-parsys/container_428510884/container/container/container/container/image_1801542987.coreimg{.width}.svg/1669130241057/linkedin.svg" data-asset="/content/dam/LeaderImpact/site-./assets/linkedin.svg" data-title="LinkedIn" id="image-bb85cd5a39" class="cmp-image" itemscope="" itemtype="http://schema.org/ImageObject">
+                              <img src="./assets/linkedin.svg" class="cmp-image__image" itemprop="contentUrl" data-cmp-hook-image="image" alt="LinkedIn">
+                            </div>
+                          </div>
                         </div>
+                      </div>
                     </div>
+                  </div>
                 </div>
+              </div>
             </div>
-        </section>
-        <section class="alert-bar">
-            <div class="container">
-                <div class="alert-logo-wrap">
-                    <div data-asset-id="" id="" data-cmp-data-layer="" class="cmp-image" itemscope="" itemtype="">
-                        <img srcset="./assets/alert-logo.svg" src="./assets/alert-logo.svg" loading="lazy" class="logo-img" itemprop="contentUrl"  alt="logo" title="logo">
-                        <span class="alert-text" itemprop="">Have an impact as a young professional.</span>
-                    </div>
+          </div>
+        </div>
+
+        <div class="container responsivegrid banner aem-GridColumn aem-GridColumn--default--12 layout-grid">
+          <div id="container-d78e99ebf0" class="cmp-container" style="background-image:url( ./assets/Banner.png);background-size:cover;background-repeat:no-repeat;">
+          <div class="container responsivegrid banner-wrap">

```

Comment:
This `<div>` needs a child element with the class `cmp-container` wrapping around the title and buttons.